{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "a9101efc_e2b6b340",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 1
      },
      "lineNbr": 0,
      "author": {
        "id": 8667
      },
      "writtenOn": "2021-11-19T16:35:12Z",
      "side": 1,
      "message": "Locally the error was fixed by these changes. Lets see what happens at the gate",
      "revId": "0d74fedb8836f6e0e14322defe63b52281219f38",
      "serverId": "7fc14799-209e-464c-9743-7a06c2c21a81"
    }
  ]
}